storage/innobase/xtrabackup/src/backup_copy.cc

```diff
@@ -1746,85 +1851,19 @@ copy_back()
 
 	it = datadir_iter_new(".", false);
```

Comment:
Sure, this is the same way `xtrabackup_backup_func` works. But sure, let me put that in the blueprint.